compiler/transforms/memref_to_snax.py

```diff
@@ -61,6 +61,8 @@ def match_and_rewrite(self, alloc_op: memref.Alloc, rewriter: PatternRewriter):
                 ops_to_add.append(shape_op)
                 shape_ops.append(shape_op)
 
+        shape_ops_arg = [x for x in shape_ops]
+
```

Comment:
This is not doing anything, right? We could just pass shape_ops immediately, no?